tasks/gulp/compile-assets.mjs

```diff
@@ -105,7 +85,15 @@ compileStylesheets.displayName = 'compile:scss'
 function compileStylesheet (stream, options = {}) {
   return stream
     .pipe(plumber(errorHandler(stream, 'compile:scss')))
-    .pipe(sass(options))
+    .pipe(sass({
+      ...options,
+
+      // Resolve @imports via
+      includePaths: [
```

Comment:
I can't work out where this directory is coming from (and I just want to make sure these changes are hitting it), do you have a clearer idea?

https://github.com/alphagov/govuk-frontend/blob/a1709a05575cdcaa9fb2342afd516e1faef0512f/app/assets/scss/app-legacy-ie8.scss#L17

Reply:
@domoscargin Yeah I'll talk it through 👍 

The `$path` variable is for building icon background URLs
https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_url-helpers.scss#L10

We add various `express.static()` middlewares via:
https://github.com/alphagov/govuk-frontend/blob/main/app/common/middleware/vendor.js

But mount to `/vendor` on this line:
https://github.com/alphagov/govuk-frontend/blob/main/app/app.js#L65

Reply:
Also worth mentioning, this PR now uses the same **includePaths** on all stylesheets

Where previously only `app-legacy?(-ie8).scss` were given them in:
https://github.com/alphagov/govuk-frontend/blob/35a4fb0ded7f0778e0e30c40876e8f3817add83b/tasks/gulp/compile-assets.mjs#L76

Now all stylesheets get the same (optional) ability to resolve imports from:

1. Packages in `node_modules`
2. GOV.UK Frontend Toolkit stylesheets

In a bit more detail:

## GOV.UK Elements via `node_modules`

Adding `node_modules` to **includePaths** makes sure this works:

```scss
// @import "node_modules/govuk-elements-sass/public/sass/govuk-elements";
@import "govuk-elements-sass/public/sass/govuk-elements";
```

But we have another problem 😮 

Looking [inside `govuk-element-sass`](https://github.com/alphagov/govuk_elements/blob/master/packages/govuk-elements-sass/public/sass/_frontend-toolkit.scss) none of these imports exist:

```scss
// Error: File to import not found or unreadable
@import "colours";
@import "font_stack";
@import "measurements";
```

So we need another `includePaths` addition!

## GOV.UK Frontend Toolkit via `node_modules/govuk_frontend_toolkit/stylesheets`

Adding `node_modules/govuk_frontend_toolkit/stylesheets` to **includePaths** resolves to:

```scss
// @import "node_modules/govuk_frontend_toolkit/stylesheets/colours";
@import "colours";

// @import "node_modules/govuk_frontend_toolkit/stylesheets/font_stack";
@import "font_stack";

// @import "node_modules/govuk_frontend_toolkit/stylesheets/measurements";
@import "measurements";
```

## GOV.UK Frontend

But by adding `node_modules` to **includePaths** (for all stylesheets) it's now incredibly useful when switching the Review app to a "release" version (temporarily) by changing:

### Before: Import from source
Import Sass from **./src**

```scss
@import "../../src/govuk/all";
 ```

### After: Import from package
Import pre-built Sass from **./package**

```scss
@import "govuk-frontend/govuk/all";
```

Thanks to `govuk-frontend` available as a symbolic link (see npm workspace `npm ls govuk-frontend`)

```console
├─┬ govuk-frontend-webpack-boilerplate@ -> ./docs/examples/webpack
│ └── govuk-frontend@4.5.0 deduped -> ./package
└── govuk-frontend@4.5.0 -> ./package
```

Reply:
OK, nice - thanks for the detailed explanation, that was really helpful (would've taken me ages to find that vendor middleware line :joy:)